services/blockchain-indexer/methods/dataService/controllers/peers.js

```diff
@@ -60,6 +64,9 @@ const getPeersStatistics = async () => {
 	if (response.data) peerStatistics.data = response.data;
 	if (response.meta) peerStatistics.meta = response.meta;
 
+	peerStatistics.data.totalLocked = await getTotalLocked();
+	peerStatistics.data.totalStaked = await getTotalStaked();
```

Comment:
Let us discuss this offline.

Reply:
will be moved under index.network.statistics as per discussion.